METIS/METIS_DET_IFU.yaml

```diff
@@ -23,12 +26,23 @@ effects:
       kwargs:
           filename: QE_detector_H2RG_METIS.dat
 
+    - name: auto_exposure
+      description: automatic determination of DIT and NDIT
+      class: AutoExposure
+      include: True
+      kwargs:
+        fill_frac: "!OBS.auto_exposure.fill_frac"
+        exptime: "!OBS.auto_exposure.exptime"
+
+    - name: exposure_action
+      description: Summing up sky signal for all DITs and NDITs
+      class: SummedExposure
+
     - name: dark_current
       description: METIS LMS dark current
       class: DarkCurrent
-      # [e-/s] level of dark current for each detector
       kwargs:
-          value: 0.1
+          value: 0.1  # [e-/s]
```

Comment:
if you're really really really strict on units it would even be [e-/s/pix]
("grammar-nazi" equivalent for units, I know ....)